cpp/include/raft/neighbors/detail/cagra/cagra_build.cuh

```diff
@@ -212,18 +215,23 @@ void build_knn_graph(raft::resources const& res,
 
     size_t num_queries_done = batch.offset() + batch.size();
     const auto end_clock    = std::chrono::system_clock::now();
-    const auto time =
-      std::chrono::duration_cast<std::chrono::microseconds>(end_clock - start_clock).count() * 1e-6;
-    const auto throughput = num_queries_done / time;
-    RAFT_LOG_DEBUG(
-      "# Search %12lu / %12lu (%3.2f %%), %e queries/sec, %.2f minutes ETA, self included = "
-      "%3.2f %%    \r",
-      num_queries_done,
-      dataset.extent(0),
-      num_queries_done / static_cast<double>(dataset.extent(0)) * 100,
-      throughput,
-      (num_queries - num_queries_done) / throughput / 60,
-      static_cast<double>(num_self_included) / num_queries_done * 100.);
+    if (batch.offset() > next_report_offset) {
+      next_report_offset += d_report_offset;
+      const auto time =
+        std::chrono::duration_cast<std::chrono::microseconds>(end_clock - start_clock).count() *
+        1e-6;
+      const auto throughput = num_queries_done / time;
+
+      RAFT_LOG_INFO(
```

Comment:
Note: I have changed the build progress messages from DEBUG to INFO. Since build takes long (10+ min for large datasetst) I find it useful to get a message about the progress. This will emit 100 message during the build process. The DEBUG messages are be flooded by other IVF-PQ debug printouts, therefore I set this to INFO.